components/encryption/src/manager/mod.rs

```diff
@@ -63,6 +64,8 @@ impl Dicts {
                 file_dict.merge_from_bytes(&file_bytes)?;
                 let mut key_dict = KeyDictionary::default();
                 key_dict.merge_from_bytes(&key_bytes)?;
+                ENCRYPTION_DATA_KEY_GAUGE.set(key_dict.keys.len() as _);
```

Comment:
can we also set `ENCRYPTION_FILE_NUM_GAUGE` here?

Reply:
But how to determine the encryption method? unless traversing ...

Reply:
okay, let's don't breakdown by encryption method then.